File: Recursion/Problem 1/p1.js
function sum(n){  

    if(n<0) return "Enter a positive number"
  
    if(n===0){
      return 0
      }else {
        return sum(n-1) + n
      }
   }  
   console.log(sum(5)) //15
   console.log(sum(28)) //406
   console.log(sum(300)) //45150

File: Recursion/Problem 1/p1.py
def sum(n){

  if(n<0) return "Enter a positive number"

  if(n===0){
    return 0
    }else {
      return sum(n-1) + n
    }
 }
 print(sum(5)) //15
 print(sum(28)) //406
 print(sum(300)) //45150